content/docs/command-reference/add.md

```diff
@@ -344,3 +342,49 @@ $ tree .dvc/cache
 
 Only the hash values of the `dir/` directory (with `.dir` file extension) and
 `file2` have been cached.
+
+## Example: Transfer to remote storage
+
+When you have a large dataset in an external location, you may want to add it to
+your project without downloading it to the local file system (for using it
+later/elsewhere). The `--to-remote` option let you skip the download, while
+storing the imported data [remotely](/doc/command-reference/remote). Let's
+initialize a DVC project, and setup a remote:
```

Comment:
Here's that example intro copy edit I promised 🙂 

```suggestion
When you have a large dataset in an external location, you may want to track it
in your project without copying (downloading) it to the local <abbr>cache</abbr>
(for now). The `--to-remote` option let you skip the download, while storing the
added data [remotely](/doc/command-reference/remote). Let's initialize a DVC
project, and setup a remote:
```

Reply:
WAIT... I'm confuse here actually... Yeah, what download are we talking about @isidentical? Isn't `add --to-remote` a form of `add --external`? Meaning that an external cache needs to be setup in the external location first.

If so, this would just skip the caching of the data, and instead upload it to some remote. But I thought external data can't be pushed to remote storage. Yeah I'm very confused now 😕 please re-explain once more...

Because otherwise what's the difference between `import-url --to-remote` and `add --to-remote`? If it's just that the data source isn't saved in the .dvc file I'd go back to questioning the need for this option in `add`.

Reply:
> If it's just that the data source isn't saved in the .dvc file

That would mean that `add --to-remote` would have the same result as `dvc get-url` + `dvc add` + `dvc push` + `dvc remove` and `dvc gc`. Is that the use case?

If so I def. think that `import-url --to-remote` is always a better option, and way less confusing 🙂

Reply:
>  Is that the use case?

exactly! I don't get `--external` thing it does the exact steps you gave as an example

Reply:
> Because otherwise what's the difference between import-url --to-remote and add --to-remote? If it's just that the data source isn't saved in the .dvc file I'd go back to questioning the need for this option in add.

Yes, that is the only difference.

Reply:
OK thanks @isidentical I'll modify my copy edits accordingly for now but I'll also open a core issue about this so you may want to wait a little bit before addressing any more of this PR 🙂

Reply:
OK so we can apply my suggestion on this one, it seems. I'll do so (which will resolve this review).